[2023-04-14 13:27:42,520] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: spotify_final_dag.create_table 2023-04-14T13:27:32.492707+00:00 [queued]>
[2023-04-14 13:27:42,661] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: spotify_final_dag.create_table 2023-04-14T13:27:32.492707+00:00 [queued]>
[2023-04-14 13:27:42,666] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 13:27:42,697] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-04-14 13:27:42,698] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-04-14 13:27:42,871] {taskinstance.py:1089} INFO - Executing <Task(PostgresOperator): create_table> on 2023-04-14T13:27:32.492707+00:00
[2023-04-14 13:27:42,895] {standard_task_runner.py:52} INFO - Started process 55 to run task
[2023-04-14 13:27:42,923] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'spotify_final_dag', 'create_table', '2023-04-14T13:27:32.492707+00:00', '--job-id', '82', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/spotify_final_dag.py', '--cfg-path', '/tmp/tmpwrgoefe6', '--error-file', '/tmp/tmpwldlyezy']
[2023-04-14 13:27:42,927] {standard_task_runner.py:77} INFO - Job 82: Subtask create_table
[2023-04-14 13:27:43,125] {logging_mixin.py:104} INFO - Running <TaskInstance: spotify_final_dag.create_table 2023-04-14T13:27:32.492707+00:00 [running]> on host 75651b8604e6
[2023-04-14 13:27:43,349] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=spotify_final_dag
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-04-14T13:27:32.492707+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-14T13:27:32.492707+00:00
[2023-04-14 13:27:43,353] {postgres.py:69} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS top_songs (
            song_date VARCHAR(200),
            song_name VARCHAR(200),
            artist_name VARCHAR(200)
            )
[2023-04-14 13:27:43,425] {taskinstance.py:1482} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1138, in _run_raw_task
    self._prepare_and_execute_task_with_callbacks(context, task)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1311, in _prepare_and_execute_task_with_callbacks
    result = self._execute_task(context, task_copy)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/taskinstance.py", line 1341, in _execute_task
    result = task_copy.execute(context=context)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/postgres/operators/postgres.py", line 71, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/hooks/dbapi.py", line 173, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/providers/postgres/hooks/postgres.py", line 83, in get_conn
    conn = self.connection or self.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/hooks/base.py", line 67, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/connection.py", line 354, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `postgres_default` isn't defined
[2023-04-14 13:27:43,432] {taskinstance.py:1532} INFO - Marking task as UP_FOR_RETRY. dag_id=spotify_final_dag, task_id=create_table, execution_date=20230414T132732, start_date=20230414T132742, end_date=20230414T132743
[2023-04-14 13:27:43,547] {local_task_job.py:146} INFO - Task exited with return code 1
